lib/src/screens/sign_up_screen.dart

```diff
@@ -0,0 +1,157 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_device_type/flutter_device_type.dart';
+
+import '../constants/constants_color.dart';
+import '../constants/constants_text.dart';
+import '../widgets/form_sign_up.dart';
+import '../widgets/logo.dart';
+
+class SignUpScreen extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    double bannerLeftPadding = MediaQuery.of(context).size.width * (25 / 375);
+    double mainTabletPadding = 0;
+    CrossAxisAlignment bannerCrossAxisAlignment = CrossAxisAlignment.start;
+    double logoTopPadding = 0;
+    double titleTopPadding = 0;
+    double titleBottomPadding = 0;
+    double formBottomPadding = 0;
+    double bannerFlex = 0;
+    double formHeight = 0;
+    String title = "";
+
+    if (Device.get().isPhone) {
+      bannerCrossAxisAlignment = CrossAxisAlignment.start;
+      logoTopPadding = MediaQuery.of(context).size.height * 60 / 812;
+      titleTopPadding = MediaQuery.of(context).size.height * 45 / 812;
+      titleBottomPadding = MediaQuery.of(context).size.height * 88 / 812;
+      formBottomPadding = 0;
+      mainTabletPadding = 0;
+      bannerFlex = MediaQuery.of(context).size.height * 285 / 812;
+      formHeight = MediaQuery.of(context).size.height * 527 / 812;
+      title = SignUpText.titleMobileText;
+    } else {
+      bannerCrossAxisAlignment = CrossAxisAlignment.center;
+      logoTopPadding = MediaQuery.of(context).size.height * 80 / 1024;
+      titleTopPadding = MediaQuery.of(context).size.height * 130 / 1024;
+      titleBottomPadding = MediaQuery.of(context).size.height * 36 / 1024;
+      formBottomPadding = MediaQuery.of(context).size.height * 172 / 1024;
+      bannerFlex = MediaQuery.of(context).size.height * 305 / 1024;
+      formHeight = MediaQuery.of(context).size.height * 547 / 1024;
+      mainTabletPadding = MediaQuery.of(context).size.width * (172 / 768);
+      title = SignUpText.titleText;
+    }
```

Comment:
this pattern looks good.
 why we need these fomulas (ex MediaQuery.of(context).size.width * (172 / 768))